tests/cli/test_cli.py

```diff
@@ -1722,17 +1722,17 @@ def test_search_time(ctx):
 @TestModule.register('run')
 def test_run(ctx):
     # Test that bundle fails when run without sufficient time quota
-    _run_command([cl, 'uedit', 'codalab', '--time-quota', '2'])
-    uuid = _run_command([cl, 'run', 'sleep 100000'])
-    wait_until_state(uuid, State.KILLED, timeout_seconds=60)
-    check_equals(
-        'Kill requested: User time quota exceeded. To apply for more quota,'
-        ' please visit the following link: '
-        'https://codalab-worksheets.readthedocs.io/en/latest/FAQ/'
-        '#how-do-i-request-more-disk-quota-or-time-quota',
-        get_info(uuid, 'failure_message'),
-    )
-    _run_command([cl, 'uedit', 'codalab', '--time-quota', ctx.time_quota])  # reset time quota
+    # _run_command([cl, 'uedit', 'codalab', '--time-quota', '2'])
```

Comment:
Uncomment this since it'll still work in Github Actions. Sorry again about the local testing issues!

Reply:
@AndrewJGaut I think the commented out part is actually pretty flaky. It seems to sometimes pass and other times fail. See https://github.com/codalab/codalab-worksheets/actions/runs/3587073727/jobs/6037163824 (latest commit on master branch).

Reply:
Yeah, that's happened for me too and the cause (in the cases I looked at) is that the timeout is a bit too aggressive. I'll increase the timeout in the PR I made to address the issues with local testing